src/test/java/com/google/sps/BigBusinessesTest.java

```diff
@@ -0,0 +1,100 @@
+import org.junit.Before;
+import org.junit.Test;
+
+@RunWith(JUnit4.class)
+public final class BigBusinessesTest {
+  private MapLocation googlesOffice = new MapLocation(40.457177, -79.916696);
+  private List<Listing> sampleListOfBusinesses = new LinkedList<>();  
+  
+  private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  private void setUpSampleDatabase() {
+    String title = "business";
+    Entity taskEntity = new Entity("BigBusinesses");
+    String[] businesses = {"McDonalds","west elm","LA Fitness"};
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    for(String business: businesses){
+      taskEntity = new Entity("BigBusinesses");
+      taskEntity.setProperty(title, business);
+      datastore.put(taskEntity);
+    }
+    assertEquals(3, datastore.prepare(new Query("BigBusinesses")).countEntities(withLimit(10)));
+  }
+
+  // To be used to see if the sample database is able to create a 
+  // local one to be used when running the command mvn test
+  @Test
+  public void testSampleDatabase() {
+    setUpSampleDatabase();
+  }
+
+  @Test
+  public void testEliminateBigBusinessesMethod() {
+    setUpSampleDatabase();
+
+    sampleListOfBusinesses.add(new Listing("LA Fitness", new MapLocation(40.457091, -79.915331), 3.9, null, "https://www.lafitness.com/Pages/Default.aspx"));
+    sampleListOfBusinesses.add(new Listing("west elm", new MapLocation(40.456279, -79.915015), 3.6, null, "https://www.westelm.com"));
+    sampleListOfBusinesses.add(new Listing("McDonalds", new MapLocation(40.459450, -79.918479), 2.6, null, "https://www.mcdonalds.com"));
+    sampleListOfBusinesses.add(new Listing("East End Brewing Company", new MapLocation(40.459391, -79.911782), 4.7, "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.eastendbrewing.com%2F&psig=AOvVaw0kX_SAlxhA09EN3cKpt5ik&ust=1593613487774000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCLDA6oDfqeoCFQAAAAAdAAAAABAD", "http://www.eastendbrewing.com/"));
+    sampleListOfBusinesses.add(new Listing("The Shiny Bean Coffee & Tea", new MapLocation(40.496328, -79.944862), 4.9, "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", "https://theshinybean.com/"));
+    sampleListOfBusinesses.add(new Listing("Weisshouse", new MapLocation(40.456684, -79.925499), 4.3, "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", "https://www.weisshouse.com/"));
+ 
+    SmallCityService testServiceUser = new SmallCityService(googlesOffice);
+    testServiceUser.setAllBusinesses(sampleListOfBusinesses);
+    
+    // These are the listings that should be removed from the sample list 
+    // when the elimnateBigBusinesses method is called 
+    sampleListOfBusinesses.remove(0);
+    sampleListOfBusinesses.remove(1);
+    sampleListOfBusinesses.remove(2);
+    testServiceUser.eliminateBigBusinesses();
```

Comment:
Slightly confused. This should be testing the 'eliminateBigBusinesses" method, but this removes the relevant businesses before that method is called.

Reply:
I just realized their is a problem with this code and I just fixed it. I forgot that when I set the sampleListOfBusinesses variable equal to the smallCityService variable, they both reference each other. So if I remove an element manually from the sampleListOfBusinesses variable, it also gets removed from the businesses variable in the SmallCityService variable businesses. 

To fix this I made another list array (expectedListOfBusinesses) to which the appropriate Listings are added to this variable first. Therefore, when the big businesses are removed from the smallCityService variable businesses, I can check that that the Listings left in the linked list are equal to the expected List variable.